src/components/p5js-components/example.js

```diff
@@ -0,0 +1,31 @@
+export default function sketch(p5) {
```

Comment:
is this example doing anything? looks different than the other p5 react components. this isn't even a react file.